### DISCUSSION TITLE
--remoteOllamaServer returns connection error

### DISCUSSION THREAD

**Discussion by WillJesperson** [danielmiessler/Fabric#518] [Q&A] [answered]
I am running a remote Ollama server, and am connected to the same network as the server. 
I had the Ollama server working smoothly for a few day, but then I decided I wanted to try fabric with another ollama model. So I installed the model and ran the command:
`echo "tell me a fun fact" | fabric --remoteOllamaServer 192.xxx.xxx.xxx --model llama3:70b -sp ai`
It would hang on this command for a while before giving me an error about connecting to the claude server, and me being out of credits. So I went into the fabric files and removed the claude API and chat-gpt API keys which I had inserted at start up. after doing this and deleting the llama3:70b model from my server I ran: 
`echo "tell me a fun fact" | fabric --remoteOllamaServer 192.xxx.xxx.xxx -sp ai` 
I was met with a "connection error" line. 
This is confusing to me because I am still able to access and use the openwebUi I have set up, but when I try to run commands through fabric things do not work. 
Any help is very welcome!

**Comment by heavenideas** [comment]
Seems like I'm running into the same issue. It will only try to use either my groq API or mistral API even if I actively remove them, and run the same command with the --remoteOllamaServer. 

The server is working correctly as I can reach it with ollama from another shell and query the model

**Reply by heavenideas** [reply, marked as answer]
Resolved my issue:

My main thing was that the first time I had tried this I was on a mac. It was all working fine because everything is installed together, both ollama and fabric.

However now I'm on windows. So I was using fabric on WSL but my ollama server was running on my normal windows... Hence the port couldn't be accessed by default from inside of WSL.

There is a way to make windows ports accessible within WSL but I opted for installing ollama inside of WSL as well because it was quicker and preferred to have the two things together for clarity.

So in my case it was user error, but hopefully this helps other people with the same issue
